rfcs/025-lineage-stage-0.md

```diff
@@ -0,0 +1,149 @@
+- Feature Name: lineage_stage_0
+- Start Date: 2021-02-22
+- RFC PR: [amundsen-io/rfcs#24](https://github.com/amundsen-io/rfcs/pull/24)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000)
+# Amundsen Lineage - Stage 0
+
+## Summary
+
+
+Currently Amundsen doesn't have a way of surfacing lineage information for tables and columns. The idea for this first iteration is to have a way to show upstream and downstream tables and columns to users through the Table Details page so they can explore the current resource's lineage as well as navigate to related resources in Amundsen.
+The first iteration is meant to be a fast implementation of the feature that we can get feedback on and improve in future iterations.
+
+## Motivation
+
+Lineage is essential to improving data discovery in Amundsen because it allows users to know where the data for a given resource is coming from as well as where this data is used downstream. 
+
+
+## Guide-level Explanation (aka Product Details)
+
+> Explain the proposal as if it was already included in Amundsen and you were teaching it to an Amundsen user. That generally means:
+> Introducing new named concepts.
+> Explaining the feature largely in terms of examples.
+> Explaining how Amundsen users should think about the feature, and how it should impact the way they use Amundsen. It should explain the impact as concretely as possible.
+> If applicable, provide deprecation warnings, or migration guidance.
+> For implementation-oriented RFCs, this section should focus on how maintainers should think about the change, and give examples of its concrete impact. For policy RFCs, this section should provide an example-driven introduction to the policy, and explain its impact in concrete terms.
+### New Concepts
+- Lineage:
+- Upstream:
+- Downstream:
+
+Those implementing Amundsen should keep in mind that this feature is meant to provide them with a way to surface their existing lineage data by calling the service containing that data from the metadata service. This iteration won't provide a model to persist lineage on neo4j, but rather a gateway to lineage data so it can be included on lineage API responses to displayed in frontend. It is also important to understand that the feature will be disabled by default and can be enable through configuration.
```

Comment:
are you going to put the service call (from Amundsen to CW or 3rd party internal service) in Lyft internal? 

Also wouldn't https://github.com/amundsen-io/amundsencommon/pull/90 surface as the model?

Reply:
Yeah, I mean in this iteration we won't be adding any relations or lineage information to the neo4j graph